### PR TITLE
Small details missing from data registry

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

File: static/docs/use-cases/data-registries.md [hunk 1]
```diff
@@ -42,6 +38,10 @@ Advantages of using a DVC **data registry** project:
 - Security: Registries can be setup to have read-only remote storage (e.g. an
   HTTP location). Git versioning of [DVC-files](/doc/user-guide/dvc-file-format)
   allows us to track and audit data changes.
+- Data as code: Improve _lifecycle management_ with versioning of simple
+  directory structures (like Git on cloud storage), without ad-hoc conventions.
+  Leverage Git and Git hosting features such as commits, branching, pull
+  requests, reviews, and even continuous deployment of ML models.
```
Comment (jorgeorpinel) [inline static/docs/use-cases/data-registries.md]: @shcheklein you wanted to rephrase this AND move it to the bottom, or should we return it to the top but with simpler wording?

File: static/docs/use-cases/data-registries.md [hunk 2]
```diff
@@ -42,6 +38,10 @@ Advantages of using a DVC **data registry** project:
 - Security: Registries can be setup to have read-only remote storage (e.g. an
   HTTP location). Git versioning of [DVC-files](/doc/user-guide/dvc-file-format)
   allows us to track and audit data changes.
+- Data as code: Improve _lifecycle management_ with versioning of simple
```
Comment (shcheklein) [inline static/docs/use-cases/data-registries.md]: simplify like this?

```
Leverage Git workflow such as commits, branching, pull requests, reviews, and even continuous deployment for your data and models lifecycle.
```

File: static/docs/use-cases/data-registries.md (hunk 1 above)
Comment (shcheklein) [reply static/docs/use-cases/data-registries.md]: I think it's fine to have it bottom, but we should simplify and clarify it

Comment (jorgeorpinel) [reply static/docs/use-cases/data-registries.md]: ApplieD

> Leverage Git workflow such as commits, branching, pull requests, reviews, and even continuous deployment for your data and models lifecycle.

From https://github.com/iterative/dvc.org/pull/869#pullrequestreview-332886790

File: static/docs/use-cases/data-registries.md [hunk 3]
```diff
@@ -42,6 +38,9 @@ Advantages of using a DVC **data registry** project:
 - Security: Registries can be setup to have read-only remote storage (e.g. an
   HTTP location). Git versioning of [DVC-files](/doc/user-guide/dvc-file-format)
   allows us to track and audit data changes.
+- Data as code: Leverage Git workflow such as commits, branching, pull requests,
+  reviews, and even CI/CD for your data and models lifecycle. Think Git for
+  cloud storage, but without ad-hoc conventions.
```
Comment (jorgeorpinel) [inline static/docs/use-cases/data-registries.md]: Merged @shcheklein but here's the text I put, in case you want to double check.

**Pull request merged**